Theme/Theme/Assets.xcassets/Colors/PrimaryCard/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
Is this change intentional?

Reply:
It auto-generates when we create a folder in the assets.